Its.Log.Monitoring/TestDiscovery.cs

```diff
@@ -21,7 +21,7 @@ internal static class TestDiscovery
                               .GetMethods(BindingFlags.Public |
                                           BindingFlags.Instance |
                                           BindingFlags.DeclaredOnly)
-                              .Where(m => !m.GetParameters().Any())
+                              .Where(m => m.GetParameters().All(p => p.HasDefaultValue))

```

Comment:
So if there's a parameter with no default value, does this mean the method is not considered a test?